trade-finance-manager-ui/component-tests/case/facility/_macros/facility_details.component-test.js

```diff
@@ -189,106 +189,106 @@ describe(component, () => {
       wrapper.expectText('[data-cy="facility-tenor"]').toRead(params.facility.dates.tenor);
     });
 
-    describe('when facility end date feature flag is not enabled', () => {
+    describe('when shouldDisplayFacilityEndDate is false', () => {
       beforeEach(() => {
-        wrapper = render({ ...params, facility: facilityWithFacilityEndDate, isTfmFacilityEndDateFeatureFlagEnabled: false });
+        wrapper = render({ ...params, facility: facilityWithFacilityEndDate, shouldDisplayFacilityEndDate: false });
       });
       it('should not render isUsingFacilityEndDate', () => {
-        wrapper.expectText('[data-cy="is-using-facility-end-date"]').notToExist();
+        wrapper.expectText('[data-cy="facility-is-using-facility-end-date"]').notToExist();
       });
 
       it('should not render facilityEndDate', () => {
-        wrapper.expectText('[data-cy="facility-end-date"]').notToExist();
+        wrapper.expectText('[data-cy="facility-facility-end-date"]').notToExist();
       });
 
       it('should not render bankReviewDate', () => {
-        wrapper.expectText('[data-cy="bank-review-date"]').notToExist();
+        wrapper.expectText('[data-cy="facility-bank-review-date"]').notToExist();
       });
     });
 
-    describe('when facility end date feature flag is enabled', () => {
+    describe('when shouldDisplayFacilityEndDate is true', () => {
       describe('when the facility has no end date information', () => {
         beforeEach(() => {
-          wrapper = render({ ...params, isTfmFacilityEndDateFeatureFlagEnabled: true });
+          wrapper = render({ ...params, shouldDisplayFacilityEndDate: true });
         });
         it('should render a dash for isUsingFacilityEndDate', () => {
-          wrapper.expectText('[data-cy="is-using-facility-end-date"]').toRead('-');
+          wrapper.expectText('[data-cy="facility-is-using-facility-end-date"]').toRead('-');
```

Comment:
same as above

Reply:
See above

Reply:
has this been refactored? if so, please resolve